src/dftbp/timedep/timeprop.F90

```diff
@@ -2241,12 +2248,23 @@ subroutine initTDOutput(this, dipoleDat, qDat, energyDat, populDat, forceDat, co
       end if
     end if
 
+    if (this%tWriteAtomEnergies) then
+      call openFile(this, atomEnergyDat, 'atomenergies.dat')
+      write(atomEnergyDat, "(A)", advance = "NO")"#             time (fs)      |"
+      write(atomEnergyDat, "(A)", advance = "NO")"   E total (H)  |"
+      write(atomEnergyDat, "(A)", advance = "NO")"   E (atom_1) (H)   |"
+      write(atomEnergyDat, "(A)", advance = "NO")"   E (atom_2) (H)   |"
+      write(atomEnergyDat, "(A)", advance = "NO")"        ...        |"
+      write(atomEnergyDat, "(A)", advance = "NO")"   E (atom_N) (H)   |"
```

Comment:
What happens if you have only one atom?

Reply:
Can we therefore make a routine to format this properly for both cases? Possibly as a separate PR to avoid holding this one up.

Reply:
Yes, we can do it in a separate PR, I'll add it to the to do.